### DISCUSSION TITLE
Schedule code for every 2 days at 1 o'clock

### DISCUSSION THREAD

**Discussion by Jobians** [pschlan/cron-job.org#186] [General]
How to schedule this to run for every 2 days at 1 o'clock in php

'schedule' => array(
                'timezone' => 'Africa/Lagos',
                'hours' => [-1],
                'mdays' => [-1],
                'minutes' => [-5],
                'months' => [-1],
                'wdays' => [-1],
            ),

**Comment by sokoya** [comment]
This is actually not an issue, you can create the interval before calling the cron-job api

**Comment by Jobians** [comment]
@sokoya How can I do it
